src/pages/msme/SubmissionCompleted.tsx

```diff
@@ -27,7 +27,8 @@ function SubmissionCompleted() {
               },
             )}
           </Text>
-          {applicationContext.state.data?.lender.external_onboarding_url ? (
+          {applicationContext.state.data?.lender.external_onboarding_url &&
+          !applicationContext.state.data?.application.borrower_accessed_external_onboarding_at ? (
```

Comment:
In what scenario could a borrower reach this page after accessing the onboarding URL?

Reply:
If they open their link again at any time